identity-apps-core/apps/authentication-portal/src/main/webapp/includes/header.jsp

```diff
@@ -29,18 +31,71 @@
 <%-- Branding Preferences --%>
 <jsp:directive.include file="branding-preferences.jsp"/>
 
-
 <%-- Extract the name of the stylesheet--%>
 <%
     String themeName = "wso2is";
+    String language = "en"; // Default language
+    Cookie[] userCookies = request.getCookies();
+    if (userCookies != null) {
+        for (Cookie cookie : userCookies) {
+            if ("ui_lang".equals(cookie.getName())) {
+                language = cookie.getValue();
+                break;
+            }
+        }
+    }
+
+    // Specify the file path
+    String filePath = application.getRealPath("/") + "/WEB-INF/classes/LanguageOptions.properties";
+
+    // Create a Map to store the language direction
+    Map<String, String> languageDirectionMap = new HashMap<>();
+
+    // Use a BufferedReader to read the file content
```

Comment:
You do not have to add comments that duplicate the code. Add comments where the code is not self explanatory.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7220/commits/9556875aff180ee4677b0816388dbe05ff09974a